sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -120,6 +120,11 @@ func resourceSumologicMonitorsLibraryMonitor() *schema.Resource {
 							Optional:     true,
 							ValidateFunc: validation.StringInSlice([]string{"AtLeastOnce", "Always", "ResultCount", "MissingData"}, false),
 						},
+						"min_data_points": {
+							Type:         schema.TypeInt,
+							Optional:     true,
+							ValidateFunc: validation.IntAtLeast(1),
```

Comment:
Do we want to validate this being less than 100 as well?

Reply:
Yes!